__tests__/app.test.js

```diff
@@ -326,3 +326,30 @@ describe("/api/reviews/:review_id/comments", () => {
       });
   });
 });
+
+describe.only("/api/comments/:comment_id", () => {
+  it("DELETE 204: responds no content", () => {
+    return request(app)
+      .delete("/api/comments/1")
+      .expect(204)
+      .then(({ body }) => {
+        expect(body).toEqual({});
```

Comment:
You can leave this as just expect(204)
if you wanted to really test it, you could do another query to check its no longer in the db, but up to you.